packages/hint-compat-api/src/utils/browsers.ts

```diff
@@ -28,6 +28,34 @@ export const filterBrowsers = (browsers: string[]): string[] => {
     });
 };
 
+export const formatSupported = (browser: string, versionAdded?: string, versionRemoved?: string): string => {
+    const name = getFriendlyName(browser);
```

Comment:
What about renaming `browser` to `browserId` and `name` to `browser`?
I think it will make it easier to know how the message would look like. Same for `formatUnsupported`.

Reply:
How about keeping `browser` and renaming `name` to `browserName`?